src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -194,7 +194,9 @@ export function CurrencySearch({
 
   // if no results on main list, show option to expand into inactive
   const filteredInactiveTokens = useSearchInactiveTokenLists(
-    filteredTokens.length === 0 || (debouncedQuery.length > 2 && !isAddressSearch) ? debouncedQuery : undefined
+    !onlyShowCurrenciesWithBalance && (filteredTokens.length === 0 || (debouncedQuery.length > 2 && !isAddressSearch))
```

Comment:
After the latest change I'm not seeing any alternatives show up
![Screen Shot 2023-02-01 at 12 29 50 ](https://user-images.githubusercontent.com/11512321/216156490-f712fc2e-66f8-4103-9ff0-462d3ffcb449.png)